app/models/repository.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Repository < ApplicationRecord
+  validates :uid, presence: true, uniqueness: true
+
+  scope :without_license, -> { where(license_name: nil) }
```

Comment:
I would suggest filtering on `spdx_id` of the license here instead, just in case